feat(deployment): document new universal on kubernetes deployment option (#1280)
kumahq/kuma-website: app/_src/production/cp-deployment/multi-zone.md

## Usage
### Set up the global control plane

{% if_version gte:2.2.x %}
The global control plane must run on a dedicated cluster (unless using "Universal on Kubernetes" mode), and cannot be assigned to a zone.

{% tip %}

> Running global control plane in "Universal on Kubernetes" mode most likely means using PostgreSQL as storage (in-memory storage is just for testing purposes).

Reviewer: ```suggestion
Running global control plane in "Universal on Kubernetes" mode means using PostgreSQL as storage instead of Kubernetes.
```

It means that failover / HA / reliability characteristics will change.
Please read [Kubernetes](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/ha-topology/) and
[PostgreSQL](https://www.postgresql.org/docs/current/high-availability.html) for more details.

> {% endtip %}

Author: @jakubdyszkiewicz can you take a look here. WDYT is better, have this tip inside the tab or outside like it is right now?

> [PostgreSQL](https://www.postgresql.org/docs/current/high-availability.html) for more details.

Author: @lahabana let me know if this is what you had in mind.

    By default, it's exposed on {% if_version lte:2.1.x %}[port 5685](/docs/{{ page.version }}/networking/networking){% endif_version %}{% if_version gte:2.2.x %}[port 5685](/docs/{{ page.version }}/production/deployment/networking/){% endif_version %}. In this example the value is `35.226.196.103:5685`. You pass this as the value of `<global-kds-address>` when you set up the zone control planes.

{% endtab %}

{% if_version gte:2.2.x %}
{% tab global-control-plane Universal on Kubernetes using Helm %}

1. Set `controlPlane.environment=universal` and `controlPlane.mode=global` in the chart (`values.yaml`).

> 1. Set `controlPlane.secrets` with database sensitive information

Reviewer 2: shouldn't this be `2.`?

Author: It auto increments:

![image](https://user-images.githubusercontent.com/2753650/230077587-7c7bf768-a34b-4cfe-babd-9d2aee810044.png)

that way I don't have to fix it manually.

Author: If you want me to I can change this to the correct numeration :D

   ```yaml
   # ...
       secrets:
         postgresDb:
           Secret: your-secret-name
           Key: POSTGRES_DB
           Env: KUMA_STORE_POSTGRES_DB_NAME
         postgresHost:
           Secret: your-secret-name
           Key: POSTGRES_HOST_RW
           Env: KUMA_STORE_POSTGRES_HOST
         postgrestUser:
           Secret: your-secret-name
           Key: POSTGRES_USER
           Env: KUMA_STORE_POSTGRES_USER
         postgresPassword:
           Secret: your-secret-name
           Key: POSTGRES_PASSWORD
           Env: KUMA_STORE_POSTGRES_PASSWORD

>    ```

Reviewer 2: would be cool to also add a template of Secret. This might not be super clear that `Key` is a key in the secret data also what is the type of secret.
```
apiVersion: v1
kind: Secret
metadata:
  name: your-secret-name
type: Opaque
data:
  POSTGRES_DB: ...
  POSTGRES_HOST_RW: ...
  KUMA_STORE_POSTGRES_USER: ...
  KUMA_STORE_POSTGRES_PASSWORD: ...
```
